recipes/purge_haplotigs/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.1" %}
+{% set build = "0" %}
+{% set tarball = "https://bitbucket.org/mroachawri/purge_haplotigs/get/minimap2.tar.gz" %}
```

Comment:
Looks like the tarball url is not versioned. Can you directly link to a tagged release please?
Also, please only specify the version as a variable. Jinja variables are for parts that change but occur at **multiple** places.

Reply:
Thank you Johannes, I've made the changes. Please let me know if there's anything else I could be doing better as I'm quite new to this.